UMA-Bots/Large-Relayer-Token-Balance-Change/src/balance.fetcher.ts

```diff
@@ -0,0 +1,34 @@
+import { providers, Contract, BigNumber } from "ethers";
+import { Interface } from "ethers/lib/utils";
+import LRU from "lru-cache";
+import { ERC20_ABI } from "./utils";
+
+export default class BalanceFetcher {
+  provider: providers.Provider;
+  private cache: LRU<string, BigNumber | string>;
+  private tokenContract: Contract;
+
+  constructor(provider: providers.Provider) {
+    this.provider = provider;
+    this.cache = new LRU<string, BigNumber | string>({

```

Comment:
```suggestion
  private cache: LRU<string, BigNumber>;
  private tokenContract: Contract;

  constructor(provider: providers.Provider) {
    this.provider = provider;
    this.cache = new LRU<string, BigNumber>({
```

As the `balance` is strictly typed as `BigNumber`, I think the `string` can be omitted